Frameworks/UIKit/StarboardXaml/LayerCoordinator.cpp

```diff
@@ -1060,11 +1061,13 @@ void LayerCoordinator::_VisualHeightChangedCallback(DependencyObject^ sender, De
 
 // ContentGravity
 ContentGravity LayerCoordinator::GetContentGravity(FrameworkElement^ element) {
-    return static_cast<ContentGravity>(element->GetValue(s_contentGravityProperty));
+    // We store as an int (rather than as a ContentGravity enum value) so we can read the values in the VS debugger at runtime
```

Comment:
sorry, changes in UIButton/LayerCoordnator

---
In reply to: [92522383](https://github.com/Microsoft/WinObjC/pull/1545#discussion_r92522383) [](ancestors = 92522383)